neuraxle/steps/flow.py

```diff
@@ -34,7 +34,7 @@
 OPTIONAL_ENABLED_HYPERPARAM = 'enabled'
 
 
-class TrainOrTestOnlyWrapper(ForceHandleOnlyMixin, MetaStepMixin, BaseStep):
+class TrainOrTestOnlyWrapper(ResumableStepMixin, ForceHandleOnlyMixin, MetaStepMixin, BaseStep):
```

Comment:
this is a symptom: the fix must be done in the MetaStepMixin and TruncableStep to have the possibility to already "maybe" resume.